datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/mirrormaker/KafkaMirrorMakerConnectorTask.java

```diff
@@ -162,7 +162,8 @@ public KafkaMirrorMakerConnectorTask(KafkaBasedConnectorConfig config, Datastrea
     LOG.info("Destination topic prefix has been set to {}", _destinationTopicPrefix);
 
     if (_isFlushlessModeEnabled) {
-      _flushlessProducer = new FlushlessEventProducerHandler<>(_producer);
+      _flushlessProducer = new FlushlessEventProducerHandler<>(_producer,
+          ReflectionUtils.createInstance(config.getCallbackStatusStrategy()));
```

Comment:
See if this ReflectionUtils can be passed to the config check itself. Since this can result in runtime exception and we will have ensure no path is swallowing the exception.